src/meltano/core/tracking/iglu-client-embedded/schemas/com.meltano/exception_context/jsonschema/1-0-0

```diff
@@ -0,0 +1,119 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a Snowplow context describing the exceptions currently being handled",
+    "self": {
+        "vendor": "com.meltano",
+        "name": "exception_context",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "definitions": {
+        "exception": {
+            "description": "An exception currently being handled",
+            "type": [
+                "object",
+                "null"
+            ],
+            "properties": {
+                "type": {
+                    "description": "The name of the type of exception",
+                    "type": "string",
+                    "maxLength": 64,
+                    "examples": [
+                        "ValueError",
+                        "TypeError",
+                        "KeyboardInterrupt",
+                        "SystemExit",
+                        "OSError"
+                    ]
+                },
+                "str_hash": {
+                    "description": "64 character hash of the string representation of the exception given by `str`",
+                    "type": "string",
+                    "maxLength": 64
+                },
+                "repr_hash": {
+                    "description": "64 character hash of the string representation of the exception given by `repr`",
+                    "type": "string",
+                    "maxLength": 64
+                },
+                "traceback": {
+                    "description": "Traceback info from the exception",
+                    "type": [
+                        "array",
+                        "null"
+                    ],
+                    "items": {
+                        "type": "object",
+                        "properties": {
+                            "path": {
+                                "description": "The path of the Python source file for this level of the traceback, relative to the location Python was installed; null if the path could not be made relative",
+                                "type": [
+                                    "string",
+                                    "null"
+                                ],
+                                "maxLength": 1024
+                            },
+                            "path_hash": {
+                                "description": "64 character hash of the absolute path of the Python source file for this level of traceback",
+                                "type": [
+                                    "string",
+                                    "null"
+                                ],
+                                "maxLength": 64
+                            },
```

Comment:
I think I would rather not collect this, personally.

Especially with the proposed basename fallback (`.../filename.py`), what do you think of removing `path_hash`?